Documentation/Contributors/VSCodeGuide/README.md

```diff
@@ -0,0 +1,55 @@
+# VSCode Guide
```

Comment:
Add a link to this from https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors#contributor-guides